scripts/build.js

```diff
@@ -0,0 +1,35 @@
+import { copyFile, mkdir } from "fs/promises";
+
+import esbuild from "esbuild";
+
+const isMain = import.meta.url === new URL(`file://${process.argv[1]}`).toString();
```

Comment:
This here is a bit of a hack and I'd like confirmation that it works on windows.